biogrid/apps/biogrid-mvp/src/app/pages/simulate-page/simulate-page.css

```diff
@@ -3,17 +3,29 @@
 }
 
 .results {
+  display: flex;
   font-family: sans-serif;
+  justify-content: center;
+  padding: 2rem;
 }
 
+.simboard-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-grow: 1;
+}
+
+
+
 .simulation {
   background-color: #f2f2f2;
   border-radius: 0.3125rem;
   font-family: sans-serif;
-  margin: auto;
+  margin: 2rem;
   padding: 1.25rem;
   text-align: center;
-  width: 56.25rem;
+  min-width: 56.25rem;
 }
 
```

Comment:
looks like the simulation chart + table right now do not fit on the grey background. Please make sure your components fit inside :)

Reply:
Added to issue #86

Reply:
Fixed with overflow-y
![Screenshot_2020-07-30_152641](https://user-images.githubusercontent.com/18239203/88965712-24e3b500-d279-11ea-863e-ec0f257aad13.jpg)

Reply:
looks great, thanks! would prefer to have a bit more space between the table and the image, can you file a bug to fix that in the future? Thanks!

Reply:
I also don't see the "Stop" button in the demo link - please make sure to test it in the UI and make sure it works as expected :)